src/components/StackNavigation/StackNavigation.tsx

```diff
@@ -0,0 +1,88 @@
+import {FC, useEffect} from "react";
+import {useNavigate} from "react-router";
+import {Note} from "types/note";
+import {ReactComponent as RightArrowIcon} from "assets/icon-arrow-next.svg";
+import {ReactComponent as LeftArrowIcon} from "assets/icon-arrow-previous.svg";
+import "./StackNavigation.scss";
+import {StackNavigationDots} from "./Dots/StackNavigationDots";
+
+interface StackNavigationProps {
+  stacks: Note[];
+  currentStack: string;
+  prevColumnStack: string | undefined;
+  nextColumnStack: string | undefined;
```

Comment:
btw you can also write:
```suggestion
  prevColumnStack?: string;
  nextColumnStack?: string;
```